MIGRATION_GUIDE.md

```diff
@@ -9,6 +9,20 @@ across different versions.
 
 ## v0.97.0 ➞ v0.98.0
 
+### *(behavior change)* Provider configuration rework
+On our road to v1, we have decided to rework configuration to address the most common issues. We have created a list of topics we wanted to address before v1. We will prepare an announcement soon. The following subsections describe the things addressed in the v0.98.0.
```

Comment:
add here the link to the roadmap entry, this rework was communicated a few months back

Reply:
Done.